manuscript/Figure4.md

```diff
@@ -369,9 +385,9 @@ end
 CairoMakie.scatter!(axC, xs .+ rand(Normal(0, 0.1), length(xs)), ys;)
 Legend(C_subfig[1,2], [MarkerElement(; marker=:rect, color = ColorSchemes.Set3_12[i]) for i in 1:length(imp_bugs)], imp_bugs)
```

Comment:
I think we maybe want to reverse the colors or the legend. Right now it's `figure: left-> right` and `legend: bottom-> up`. I feel like left-> right, top-> down is more intuitive. Might be as simple as `... for i in eachindex(imp_bugs)], reverse(imp_bugs)...`